### PR TITLE
device: set max_work_item_sizes for host.

### PR DESCRIPTION
**Pull request by airlied**
This should fix the ParallelSTL breakage

### CODE REVIEW

File: include/CL/sycl/device/detail/host_device.hpp [hunk 1]
```diff
@@ -102,6 +102,7 @@ class host_device : public detail::device,
   get_info(info::device param) const override {
     switch (param) {
     TRISYCL_DEFINE_DEVICE_HOST_INFO_TEMPLATE(max_work_group_size, static_cast<std::size_t>(8))
+    TRISYCL_DEFINE_DEVICE_HOST_INFO_TEMPLATE(max_work_item_sizes, (cl::sycl::id<3>{ 1, 1, 1 }))
```
Comment (keryell) [inline include/CL/sycl/device/detail/host_device.hpp]: We used to have:
```
template <>
inline auto device::get_info<info::device::max_work_item_sizes>() const {
  return cl::sycl::id<3> { 128, 128, 128 };
}
```
which allows more parallelism with OpenMP.

Then I realize that we had also `max_work_group_size` of 8, which is inconsistent. :-(
It should be something between `128*1*1` and `128*128*128`.

Comment (airlied) [reply include/CL/sycl/device/detail/host_device.hpp]: Let's put back the original for now and we can come up with better values once the regression if fixed, pushed v2 with 128 in there now.

**Comment by airlied** [comment]
I think it lost my comment, but lets put back the original values we can decide if they are correct once the regression is fixed

**Comment by keryell** [comment]
Thank you for the fix.
Yes, let's discuss on the current weird value of `max_work_group_size` somewhere else.

**Pull request merged**